### DISCUSSION TITLE
When do I need to run the migrate command?

### DISCUSSION THREAD

**Discussion by usamashafiqs** [appwrite/appwrite#4683] [Q&A] [answered]
Did i need to run migration from upgrading to 1.0.3 to 1.1.0 ?

**Comment by stnguyen90** [comment, marked as answer]
Yes, all non-patch upgrades require running a migration.

**Reply by usamashafiqs** [reply]
I upgrade to v1.1.0 and did not run the migration but appwrite running just fine ?

**Reply by stnguyen90** [reply]
There may be some pieces of Appwrite that aren't working 100% without the migration.

**Reply by usamashafiqs** [reply]
Thx
